lib/acts_as_taggable_on_mongoid/default_parser.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module ActsAsTaggableOnMongoid
```

Comment:
Parsers and parsing were only done because they already existed with little to no changes on my part in ActsAsTaggableOn and to take them out was far more work than copying them over.